views/analysis.py

```diff
@@ -434,10 +434,16 @@ def layer_tiles(request):
     if request.method != 'GET':
         return HttpResponseBadRequest()
     layer_id = request.GET.get('layer_id')
+    if request.GET.get('target_srid'):
+        target_srid = request.GET.get('target_srid')
+    else:
+        target_srid = 'EPSG:4326'
     if not layer_id:
         return HttpResponseBadRequest()
     try:
         layer = Layer.objects.get(id=layer_id)
+        if layer.srid != target_srid:
+            layer.bbox_x0, layer.bbox_y0, layer.bbox_x1, layer.bbox_y1 = transform_layer_bbox(layer, target_srid)
```

Comment:
Assign this to local variables like: `bbox_x0, bbox_y0, bbox_x1, bbox_y1`.
This is to prevent reader from being misled that the value comes directly from the layer.